databases/week2/part_3.sql

```diff
@@ -0,0 +1,52 @@
+-- Active: 1660903376490@@127.0.0.1@3306@hyf_lession_2
+
+--Part 3: More queries
+
+USE hyf_lession_2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT
+    task.title AS 'Task Name',
+    user.email AS 'email'
+FROM user_task
+    JOIN task ON user_task.task_id = task.id
+    JOIN user ON user_task.user_id = user.id
+WHERE
+    user.email LIKE '%@spotify.com%';
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+
+SELECT
+    task.title AS 'Task Name',
+    user.name AS 'user',
+    status.name AS status
+FROM user_task
+    JOIN task ON user_task.task_id = task.id
+    JOIN user ON user_task.user_id = user.id
+    JOIN status ON task.status_id = status.id
+WHERE
+    status.id = 1
+    AND user.name = 'Donald Duck';
```

Comment:
Small modification...better and everyone can easily understand when using `status. name` instead of `status.id=1`
```
SELECT task.id,
    task.title AS 'Task Name',
    user.name AS 'user',
    status.name AS status
FROM user_task
    JOIN task ON user_task.task_id = task.id
    JOIN user ON user_task.user_id = user.id
    JOIN status ON task.status_id = status.id
WHERE
    status.name = "Not started"
    AND user.name = 'Donald Duck';
```

Reply:
Yes will update it